presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -355,3 +360,67 @@ Lambda Aggregate Functions
         -- (2, 42)
 
     The state type must be a boolean, integer, floating-point, or date/time/interval.
+
+
+Filtering During Aggregation
+----------------------------
+
+The ``FILTER`` keyword can be used to remove rows from aggregation processing
+with a condition expressed using a ``WHERE`` clause. This is evaluated for each
+row before it is used in the aggregation and is supported for all aggregate
+functions.
+
+    AGGREATE_FUNCTION(...) FILTER(WHERE <condition>)
+
+For example, you want to add a condition on the count for Iris flowers,
+modifying the following query.
+
+.. code-block:: none
```

Comment:
The canonical example (and very useful example) is to use `FILTER` to remove nulls from consideration when using `array_agg`:

```
SELECT array_agg(col) FILTER (WHERE col IS NOT NULL)
FROM t
```

There's no easy way to achieve that, especially when the query involves other aggregations.